UpdatedSyntheticDataset/SyntheticDataset2/ImageOperations/image_paster.py

```diff
@@ -0,0 +1,96 @@
+from PIL import Image
+import math
+from SyntheticDataset2.ElementsCreator import *
+from SyntheticDataset2.ImageOperations import *
```

Comment:
Imports should be relative not module imports. I know the old code is like this, but I didn't realize how much of a problem this is in complex systems until recently. I apologize for my bad. If you have questions we can talk about it tomorrow in class.